README.md

```diff
@@ -82,38 +82,23 @@ export PATH=$PATH:$GOPATH/bin
 
 #### Download and Compile IPFS
 
-go-ipfs differs from the vanilla `go get` flow: it uses
-[gx](https://github.com/whyrusleeping/gx)/[gx-go](https://github.com/whyrusleeping/gx-go)
```

Comment:
Reduces cognitive overload. The entire install process doesn't depend on whether the user/dev knows `gx` is used under the hood or not.